datajunction-server/datajunction_server/sql/functions.py

```diff
@@ -1195,29 +1215,37 @@ def infer_type(
     return ct.DateType()
 
 
-class DateDiff(Function):
+class Datediff(Function):
```

Comment:
Having it as `DateDiff` meant that we expect the spark function to be `date_diff`, but the actual function is `datediff` without the underscore.